### PR TITLE
ADEN-9361 Add new SSP params for Futhead

### PR DESCRIPTION
**Pull request by dmnsobczak**

### CODE REVIEW

File: platforms/sports/bidders/prebid/app-nexus.ts [hunk 1]
```diff
@@ -1,55 +1,103 @@
...
-			enabled: false,
-			slots: {
-				'01_LB': {
-					sizes: [[728, 90], [970, 250]],
-					placementId: '16755870',
+		muthead: {
+			desktop: {
+				enabled: false,
+				slots: {
+					'01_LB': {
```
Comment (nandy-andy) [inline platforms/sports/bidders/prebid/app-nexus.ts]: Why in the spreadsheet we have more placement ids than in this config?

File: platforms/sports/bidders/prebid.ts [hunk 2]
```diff
@@ -6,13 +6,17 @@ import { getOpenXContext } from './prebid/openx';
 import { getPubmaticContext } from './prebid/pubmatic';
 import { getRubiconContext } from './prebid/rubicon';
 
-export function setPrebidAdaptersConfig(): void {
+export function setPrebidAdaptersConfig(app: string): void {
```
Comment (nandy-andy) [inline platforms/sports/bidders/prebid.ts]: That's unfortunate. We'll need to revert it once we have two separate platforms for Muthead and Futhead.

File: platforms/sports/bidders/prebid.ts [hunk 3]
```diff
@@ -6,13 +6,17 @@ import { getOpenXContext } from './prebid/openx';
...
-	context.set('bidders.prebid.openx', getOpenXContext(mode));
-	context.set('bidders.prebid.pubmatic', getPubmaticContext(mode));
-	context.set('bidders.prebid.rubicon_display', getRubiconContext(mode));
+	context.set('bidders.prebid.appnexus', getAppNexusContext(app, mode));
+	context.set('bidders.prebid.indexExchange', getIndexExchangeContext(app, mode));
+	context.set('bidders.prebid.openx', getOpenXContext(app, mode));
+	context.set('bidders.prebid.pubmatic', getPubmaticContext(app, mode));
 	context.set('bidders.prebid.wikia', getWikiaContext(mode));
+
+	// Temporary till we get SSP params for Rubicon
```
Comment (nandy-andy) [inline platforms/sports/bidders/prebid.ts]: `r/Temporary till we get SSP params for Rubicon/Temporary till we get SSP params for Futhead from Rubicon/`

File: platforms/sports/setup-context.ts [hunk 4]
```diff
@@ -46,16 +46,16 @@ class ContextSetup {
...
-		setPrebidAdaptersConfig();
-		setupBidders(context, this.instantConfig);
-
 		this.instantConfig.isGeoEnabled('wgAdDriverLABradorTestCountries');
 
 		context.set('slots', slotsContext.generate());
 		context.set('targeting', getPageLevelTargeting());
 		context.set('options.maxDelayTimeout', this.instantConfig.get('wgAdDriverDelayTimeout', 2000));
 
+		setA9AdapterConfig();
```
Comment (nandy-andy) [inline platforms/sports/setup-context.ts]: What's the reason it was moved here?

File: platforms/sports/bidders/prebid.ts (hunk 2 above)
Comment (dmnsobczak) [reply platforms/sports/bidders/prebid.ts]: Yep :< We discussed that with @Bielik20 offline and we decided that we can live with it for some time.

File: platforms/sports/bidders/prebid/app-nexus.ts (hunk 1 above)
Comment (dmnsobczak) [reply platforms/sports/bidders/prebid/app-nexus.ts]: Discussed offline

File: platforms/sports/setup-context.ts (hunk 4 above)
Comment (dmnsobczak) [reply platforms/sports/setup-context.ts]: `context.set('targeting', getPageLevelTargeting());` need to be executed first in order to save `targeting.s1` to context.

File: platforms/sports/bidders/prebid.ts (hunk 3 above)
Comment (dmnsobczak) [reply platforms/sports/bidders/prebid.ts]: Done!

**Pull request merged**